dpnp/tests/test_random_state.py

```diff
@@ -637,20 +625,19 @@ def test_distr(self, usm_type):
                 dtype=dtype,
             )
 
-        # TODO: discuss with opneMKL: there is a difference between CPU and GPU
+        # TODO: discuss with oneMKL: there is a difference between CPU and GPU
         # generated samples since 9 digit while precision=15 for float64
-        # precision = dpnp.finfo(dtype=numpy.float64).precision
-        precision = dpnp.finfo(dtype=numpy.float32).precision
-        assert_array_almost_equal(data.asnumpy(), expected, decimal=precision)
+        precision = dpnp.finfo(numpy.float32).precision
```

Comment:
please leave that as part of TODO comment
```suggestion
        # precision = dpnp.finfo(numpy.float64).precision
        precision = dpnp.finfo(numpy.float32).precision
```